Doc/library/math.rst

```diff
@@ -247,6 +249,26 @@ Floating point arithmetic
    .. versionadded:: 3.13
 
 
+.. function:: fmax(x, y, /)
+
+   Get the larger of two floating-point values, treating NaNs as missing data.
+
+   When both operands are NaNs, return ``nan`` (the sign of the result is
+   implementation-defined).
+
+   .. versionadded:: next
+
+
+.. function:: fmin(x, y, /)
+
+   Get the smaller of two floating-point values, treating NaNs as missing data.
+
+   When both operands are NaNs, return ``nan`` (the sign of the result is
```

Comment:
I'm not entirely sure that -0.0 and 0.0 are treated with their sign. It might also be implementation defined but I'll need to recheck the standard.